mesonbuild/mintro.py

```diff
@@ -124,18 +124,43 @@ def list_target_files(target_name, coredata, builddata):
 
 def list_buildoptions(coredata, builddata):
     optlist = []
-    add_keys(optlist, coredata.user_options)
-    add_keys(optlist, coredata.compiler_options)
-    add_keys(optlist, coredata.base_options)
-    add_keys(optlist, coredata.builtins)
+
+    dir_option_names = ['prefix',
+                        'libdir',
+                        'libexecdir',
+                        'bindir',
+                        'sbindir',
+                        'includedir',
+                        'datadir',
+                        'mandir',
+                        'infodir',
+                        'localedir',
+                        'sysconfdir',
+                        'localstatedir',
+                        'sharedstatedir']
```

Comment:
maybe sort these lists alphabetically?

Reply:
Sure. Fixed with the last commit.
I also sorted the lists in `mconf.py` since I have copied them from there.